assets/javascripts/modules/autoComplete.js

```diff
@@ -4,7 +4,7 @@ require('jquery');
 
 Suggestions AutoComplete module
```

Comment:
Remove "Suggestions" from the comments so it reads "AutoComplete" (as that's a more generic name for the module)